diff -ur a/SCons/Tool/__init__.py b/SCons/Tool/__init__.py
--- a/SCons/Tool/__init__.py	2020-03-22 00:59:45.410000000 +1300
+++ b/SCons/Tool/__init__.py	2020-03-21 16:20:04.370000000 +1300
@@ -230,7 +230,8 @@
             sys_modules_value = sys.modules.get(found_name, False)
 
             found_module = None
-            if sys_modules_value and sys_modules_value.__file__ == spec.origin:
+            # str4d: Modified to handle __file__ not existing
+            if sys_modules_value and hasattr(sys_modules_value, '__file__') and sys_modules_value.__file__ == spec.origin:
                 found_module = sys.modules[found_name]
             else:
                 # Not sure what to do in the case that there already
